config/initializers/govspeak_sanitizer.rb

```diff
@@ -0,0 +1,16 @@
+# Govspeak sanitizer appears completely unconfigurable so resorting to monkey patching
+
+module TariffGovspeakSanitizer
+  def sanitize_config(...)
+    orig_config = super
+
+    Sanitize::Config.merge(
+      orig_config,
+      attributes: {
+        'a' => orig_config[:attributes]['a'] + %w[target],
+      },
+    )
+  end
+end
+
+Govspeak::HtmlSanitizer.prepend TariffGovspeakSanitizer
```

Comment:
monkey patching is a last resort, wondering if we cannot just replace that piece of content (not very elegant, but maybe less bad than monkey patching (and easier to understand).